steps/terraform.yaml

```
@@ -0,0 +1,149 @@
+parameters:
+  - name: initCommandOptions
+    default: ''
+
+  - name: planCommandOptions
+    default: ''
+
+  - name: applyCommandOptions
+    default: ''
+
+  - name: destroyCommandOptions
+    default: ''
+
+  - name: environment
+    default: sandbox
+
+  - name: location
+    default: 'UK South'
+    values:
+      - 'UK South'
+      - 'UK West'
+
+  - name: overrideAction
+    default: plan
+    values:
+      - plan
+      - apply
+      - destroy
+
+  - name: product
+    default: ''
+
+  - name: stack
```

component is what we normally use, let's retire stack I think
```suggestion
  - name: component
```